docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,576 +1,641 @@
-    | `Octahedron`      | Equivalent to a beveled cube with a bevel percentage of 1. |
-    | `BevelledCube`    | A cube/cuboid with all edges flattened to some degree. |
-
-**Usage:**
-
-The following example illustrates how to use a property map for a primitive visual. The visual is created with the factory `CreateVisual()` method.
-
-```
-public int Shape
-{
-    get
-    {
-        return _shape;
-    }
-    set
-    {
-        _shape = value;
-
-        /// Create and register the primitive visual
-        PropertyMap primitiveVisual = new PropertyMap();
-        primitiveVisual.Add(Visual.Property.Type, new PropertyValue((int)Visual.Type.Primitive))
-                       .Add(PrimitiveVisualProperty.Shape, new PropertyValue(_shape))
-                       .Add(PrimitiveVisualProperty.BevelPercentage, new PropertyValue(0.3f))
-                       .Add(PrimitiveVisualProperty.BevelSmoothness, new PropertyValue(0.0f))
-                       .Add(PrimitiveVisualProperty.ScaleDimensions, new PropertyValue(new Vector3(1.0f, 1.0f, 0.3f)))
-                       .Add(PrimitiveVisualProperty.MixColor, new PropertyValue(new Vector4((245.0f/255.0f), (188.0f/255.0f), (73.0f/255.0f), 1.0f)));
-        _primitiveVisual = VisualFactory.Get().CreateVisual(primitiveVisual);
-        RegisterVisual(PrimitiveVisualPropertyIndex, _primitiveVisual);
-
-        /// Set the primitive visual depth index
-        _primitiveVisual.DepthIndex = PrimitiveVisualPropertyIndex;
-    }
-}
-```
-
-### Primitive Visual Examples
+Examples of primitives are presented below.
 
 - **Sphere:**
 
     ![Sphere](media/sphere.png)
 
 - **Conics:**
 
-    | Frustum                                | Cone                    | Cylinder                        |
+    | Frustrum                                | Cone                    | Cylinder                        |
     |---------------------------------------|-----------------------|-------------------------------|
-    | ![Frustum](media/conical-frustrum.png) | ![Cone](media/cone.png) | ![Cylinder](media/cylinder.png) |
+    | ![Frustrum](media/conical-frustrum.png) | ![Cone](media/cone.png) | ![Cylinder](media/cylinder.png) |
 
 - **Bevel:**
 
-   The bevel percentage defines the ratio of the outer face widths to the overall cuboid width, based on the shape's smallest dimension. Its range is 0.0 to 1.0.
+   Figures for different value of the `BevelPercentage` property.
 
-    | 0.0 (cube)                               | 0.3                                      |
-    |----------------------------------------|----------------------------------------|
-    | ![Cube](media/cube.png)                  | ![Low bevel](media/bevelled-cube-low.png) |
-    | **0.7** | **1.0 (octahedron)** |
-    | ![High bevel ](media/bevelled-cube-high.png) | ![Octahedron](media/octahedron.png)      |
+    | 0.0 (cube)                                  | 0.3                                       |
```

Comment:
Is the figure of the octahedron fine?  verify, please

Reply:
it looks like this because it is lit in such a specific way.

as I pointed out at the beginning, I would prefer at this stage to leave the drawings of primitives from the old version of the guide. after I clarify with the developers of the source code some issues, I will post another pr.